Makefile

```diff
@@ -69,6 +73,10 @@ GITHUB_TOKEN ?= ""
 
 GOBORING ?= ""
 
+## Tool Versions
+
+CONTROLLER_TOOLS_VERSION ?= "v0.9.2"
```

Comment:
Should we use the latest one - https://github.com/kubernetes-sigs/controller-tools/releases

Reply:
Done